Sources/SwiftFormatWhitespaceLinter/WhitespaceLinter.swift

```diff
@@ -414,21 +414,21 @@ public class WhitespaceLinter {
 
 extension Diagnostic.Message {
   static let trailingWhitespaceError = Diagnostic.Message(
-    .warning, "[TrailingWhitespace]: remove trailing whitespace.")
+    .warning, "[TrailingWhitespace]: remove trailing whitespace")
 
   static func indentationError(_ spaces: Int) -> Diagnostic.Message {
-    return .init(.warning, "[Indentation]: indentation should be \(spaces) spaces.")
+    return .init(.warning, "[Indentation]: indentation should be \(spaces) spaces")
```

Comment:
Should we make this one and the one below imperative? We'd need to change the API to take the actual and expected values so that we can do the math and say "indent/unindent by N spaces" and "add/remove N spaces" instead of the indicative mood message we have right now.

If we do it I think it should be a separate PR, I'm just curious about your thoughts.

Reply:
I think that would be a good change. It would be consistent with the missing new lines error and is slightly more specific. Given that it's a more substantial change than rewording, I also think a separate PR makes sense.